lib/perl/Genome/Model/Tools/Vcf/EvaluateVcfs.t

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env genome-perl
+
+# P R A G M A S ###############################################################
+use strict;
+use warnings;
+
+# M O D U L E S ###############################################################
+use Test::More;
+use Test::Number::Delta within => 1e-4;
+use Path::Class;
+use YAML::XS;
+use above 'Genome';
+
+# M A I N #####################################################################
+
+#This is a bare minimum test that just compiles Perl and the UR class.
+use_ok('Genome::Model::Tools::Vcf::EvaluateVcfs');
+
+# This directory contains the test data set (based from BIO-1176)
+my $basedir = Path::Class::Dir->new(
+    '/gscmnt/gc2801/analytics/idas',
+    'jira/BIO-1387/cle-data'
+);
+
+SKIP: {
+    skip "this is a long-running test & must be run within TGI", 75;
```

Comment:
Would it be possible to test some parts of this command, even if we have to skip running the `EvaluateVcf` command itself?  `long-running test` is another way of saying "test that will never be run"!